systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -669,7 +673,7 @@ DeploymentBuilder defaultCLusterOperator(String namespace, String operationTimeo
         clusterOperator.getSpec().getTemplate().getSpec().getContainers().get(0).setEnv(envVars);
 
         return new DeploymentBuilder(clusterOperator)
-                .withApiVersion("apps/v1")
+                .withApiVersion("extensions/v1beta1")
```

Comment:
@scholzj In kube v1.11 I think Deployments are available apps/v1. I know that out existing Fabric8 calls are still using extensions. We should change that, right? 

@stanlyDoge Obviously for this PR it's OK to use extensions if it's needed (though I wonder how it worked before).